### PR TITLE
Add top level nav to texts display - Desktop only

### PR DESCRIPTION
**Pull request by irina060981**
For https://github.com/alpheios-project/alpheios_nemo_ui/issues/69

As there is not configured that it is the same as for the main nav - I have created its own login template.
But I think - we could make it generic for this and for the main nav in this issue: https://github.com/alpheios-project/alpheios_nemo_ui/issues/76

image

image

### CODE REVIEW

File: alpheios_nemo_ui/data/assets/images/alpheios-logo.svg [hunk 1]
```diff
@@ -0,0 +1,7 @@
+<svg version="1.1" width="69.8px" height="50.9px" viewBox="0 0 69.8 50.9" xmlns="http://www.w3.org/2000/svg">
```
Comment (kirlat) [inline alpheios_nemo_ui/data/assets/images/alpheios-logo.svg]: It's probably better to remove `width` and `height` and set them with external CSS rules (i.e. in a CSS/Sass file). With it, we'll be able to use the same SVG file in different locations, in various sizes.

File: alpheios_nemo_ui/data/templates/alpheios/icons/Down-caret.svg [hunk 2]
```diff
@@ -0,0 +1,3 @@
+<svg width="19" height="19" fill="none" version="1.1" viewBox="0 0 19 19" xmlns="http://www.w3.org/2000/svg">
```
Comment (kirlat) [inline alpheios_nemo_ui/data/templates/alpheios/icons/Down-caret.svg]: Also, I think it's better not to hardcode width and height inside an SVG file

File: alpheios_nemo_ui/data/assets/images/alpheios-logo.svg (hunk 1 above)
Comment (irina060981) [reply alpheios_nemo_ui/data/assets/images/alpheios-logo.svg]: Yes, it is set inside css rules.
But first it loads without any css - and it looks like the following on a screeb:
   - first this logo stretches for all the screen
   - then it gets size from css rules

After I have reloaded the page a dozen times - my eyes were tied with these jumps :)
this width and height have no css priority at all - so it couldn't make any problems for css - but this decreases "image jumps on load"

Comment (kirlat) [reply alpheios_nemo_ui/data/assets/images/alpheios-logo.svg]: Oh, that makes sense! Totally agree with your approach then. 👍

Comment (balmas) [reply alpheios_nemo_ui/data/assets/images/alpheios-logo.svg]: > Yes, it is set inside css rules.
> But first it loads without any css - and it looks like the following on a screen:
> 
> * first this logo stretches for all the screen
> * then it gets size from css rules
> 
> After I have reloaded the page a dozen times - my eyes were tied with these jumps :)
> this width and height have no css priority at all - so it couldn't make any problems for css - but this decreases "image jumps on load"

oh! Is this the cause for #89  do you think?

Comment (irina060981) [reply alpheios_nemo_ui/data/assets/images/alpheios-logo.svg]: Yes, I read that issue after all my PRs and I think that it is the way css files are uploaded via flask,
first all HTML layout is loaded and only then css/jss, and css are applied after it is loaded completely.

Comment (balmas) [reply alpheios_nemo_ui/data/assets/images/alpheios-logo.svg]: ok, thanks. I will note this in the other issue.

**Pull request merged**